release-tools/src/main/java/org/springframework/data/release/jira/JqlQuery.java

```diff
@@ -66,7 +65,7 @@ public static JqlQuery from(TrainIteration iteration) {
 
 		List<String> parts = new ArrayList<>();
 
-		for (ModuleIteration module : iteration.getModulesExcept(BUILD)) {
+		for (ModuleIteration module : iteration.getModulesExcept(BUILD, ENVERS)) {
```

Comment:
I wonder if we rather should detect the module not being a JIRA managed one here than listing the ones who aren't explicitly.